cmd/commands/import.go

```diff
@@ -17,9 +17,9 @@ func NewImportCommand(c *Context) *cobra.Command {
 	)
 
 	importCmd := &cobra.Command{
-		Use:   "import <secret-key>",
+		Use:   "import <flags>",
```

Comment:
I think you should enclose `flags` around `[]` as squared brakets are usually used for optional parameters in CLI utilities

Reply:
Done